### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by leifle** [Tzahi12345/YoutubeDL-Material#1128]
**Describe the bug**
Since last night YoutTube seams to think I am a bot and no downloads are possible. In the log of my docker installation I get the following error:
ERROR: [youtube] xxxx: Sign in to confirm you’re not a bot. This helps protect our community. Learn more

**To Reproduce**
1. Copy URL into the URL filed: https://www.youtube.com/watch?v=OH0GrOaJ0NY
2. Press Download (Selecting resolution does not change anything!)

**Expected behavior**
Download to start as it did  the last year.

**Screenshots**
Error for https://www.youtube.com/watch?v=OH0GrOaJ0NY
Error while retrieving info on video with URL https://www.youtube.com/watch?v=OH0GrOaJ0NY with the following message: Error: Command failed with exit code 1: appdata/bin/yt-dlp https://www.youtube.com/watch?v=OH0GrOaJ0NY -o users/le/video/%(title)s.mp4 --write-info-json -S res:144 --merge-output-format mp4 --write-thumbnail --no-clean-info-json -j --dump-json ERROR: [youtube] OH0GrOaJ0NY: Sign in to confirm you’re not a bot. This helps protect our community. Learn more ERROR: [youtube] OH0GrOaJ0NY: Sign in to confirm you’re not a bot. This helps protect our community. Learn more

**Environment**
Installation details:
Installed version: v4.3.2 -   You are up to date.
Installation type: docker
Docker tag: nightly
Commit hash: 58f4313
Build date: 2024-07-16

This used to work until last night:
tzahi12345/youtubedl-material:nightly-2024-05-28

**Additional context**
https://www.youtube.com/watch?v=OH0GrOaJ0NY
Is just one example as all the others I tried.
The only thing that changed is that on the server some unused software was removed. But in the docker-compose I can not see that anything uses functions of the host OS, nor can I see that there is a connection to the host OS. No privileged mode or ports via the network are used. Nothing connects to the host network via docker.
The setup is traefik as reverse proxy and that part seams to work fine and was not changed since more then a year.

Basically the question is if there is a way to authenticate towards YouTube. I am a payed YouTube user and don't mind showing who I am. The only reason why I download sometimes is because quite a lot of films I am interested in disappear from YouTube lately, because people close their account or whatever their reasons are.

**Comment by lzpreslav** [comment]
https://github.com/yt-dlp/yt-dlp/issues/10128
